content/04.study.md

```diff
@@ -424,6 +424,92 @@ summarized above also apply to interfacial contact prediction for protein
+#### MHC-peptide binding
+
+An important type of PPI involves the immune system's ability to recognize the body's own cells.
+The major histocompatibility complex (MHC) plays a key role in regulating this process by binding antigens and displaying them on the cell surface to be recognized by T cells.
+Due to its importance in immunity and immune response, peptide-MHC binding prediction is an important problem in computational biology.
+Pan-allelic MHC-peptide binding prediction is a useful area of research as well, due to the highly polymorphic nature of the MHC-encoding gene region.
+Several approaches have shown promise towards both of these tasks.
+
+Shallow, feed-forward neural networks are competitive methods and have made progress toward pan-allele and pan-length peptide prediction.
+NetMHC [@doi:10.1110/ps.0239403 @doi:10.1093/bioinformatics/btv639] used sequence alignment to train a shallow neural network on variable length peptides in order to predict allele-specific binding.
+All possible deletions and insertions were applied to peptides of lengths other that 9, and the best predictive score of these was taken as optimal.
+NetMHCpan [@doi:10.1007/s00251-008-0341-z @doi:10.1186/s13073-016-0288-x] used a pseudo-sequence representation of the MHC class I molecule which included only polymorphic peptide contact residues.
+The sequences of the peptide and MHC were then represented using both sparse vector encoding and Blosum encoding, in which amino acids are encoded by matrix score vectors.
+A shallow neural network was trained on the encoded peptide sequence and MHC pseudosequence.
+Because of NetMHCpan's unique selection of an MHC amino acid sequence, it is the most suitable method covered here to make peptide affinity predictions for uncharacterized or poorly characterized MHC.
+In recent updates [@doi:10.1186/s13073-016-0288-x], NetMHCpan has been expanded to make use of the gapped sequence alignment characteristic of NetMHC, [@doi:10.1093/bioinformatics/btv639] which allows NetMHCpan-3.0 to handle both many alleles and many peptide lengths.
+MHCflurry [@doi:10.1101/054775] transforms peptide sequences using a NetMHC-type length adjustment before encoding each amino acid using a corresponding learned vector representation.
+In training the MHCflurry feed-forward neural network, the authors impute missing MHC-peptide binding affinities using a Gibbs sampling method, showing that imputation improves performance for data-sets with roughly 100 or fewer training examples.
+MHCflurry's imputation method increases its performance on poorly characterized alleles, making it competitive with NetMHCpan for this task.
+
+Using existing methods to represent uniform-length peptides, deep and mixed approaches have also shown promise towards prediction.
+Kuksa et al. [@doi:10.1093/bioinformatics/btv371] pre-trained a higher-order neural network (HONN) using a semi-restricted Boltzmann machine and compared performance with a deep neural network (DNN).
```

Comment:
I'm not familiar with HONN, we may need to introduce that in a phrase.  Is this highlight from this method that a DNN doesn't actually give state of the art performance?  Maybe that is what we want to focus on.